nconf/args.go

```diff
@@ -97,6 +95,17 @@ func (args *RootArgs) MustSetup(config interface{}, serviceName, version string)
 	return logger
 }
 
+func (args *RootArgs) loadDefaultConfig() (*RootConfig, error) {
+	c := &RootConfig{
+		Log: DefaultLoggingConfig(),
+	}
+
+	if err := args.load(c); err != nil {
+		return nil, errors.Wrap(err, "Failed to load the logging configuration")
```

Comment:
This is using the input from the config file that is passed in. So if you've got invalid yaml/json/env then it would tank. This surfaces that to the `Setup` and then `MustSetup` will panic on that error